server/controllers/trialbalance.js

```diff
@@ -187,7 +187,7 @@ function checkDebtorCreditorExists(transactions) {
     // if nothing is returned, skip error report
     if (!rows.length) { return; }
 
-    // returns a promise error report
+    // returns a error report
     return createErrorReport('ERROR.ERR_MISSING_DEBTOR_CREDITOR', true, rows);
   });
 }
```

Comment:
For something like an internal transfer (gain or loss on exchange, for example), an electronic transfer (such as a bank transfer), or a balancing transaction, it will not have a document number.  So it really only makes sense to refer to documents that concern two different parties, such as the hospital and a debtor/creditor.

That being said, we should figure out when we want to use a inv_po_id versus a doc_num.  I have a feeling that these should be one field, rather than two.